Doc/library/gettext.rst

```diff
@@ -257,7 +257,7 @@ are the methods of :class:`!NullTranslations`:
 
    .. method:: info()
 
-      Return the "protected" :attr:`_info` variable, a dictionary containing
+      Return the "protected" :attr:`!_info` variable, a dictionary containing
```

Comment:
Should we even be mentioning this attribute at all, if users aren't supposed to know about it?! Maybe we could just say this:

```suggestion
      Return a dictionary containing
```